dynamodb-local/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM groovy:jdk8-alpine AS build-env
+
+USER root
+RUN apk add --no-cache alpine-sdk subversion swig bash
+
+# install gant
+USER groovy
+RUN mkdir .gradle && echo "gant_installPath = \${System.properties.'user.home'}/lib/gant" > .gradle/gradle.properties
+RUN curl -sL https://github.com/Gant/Gant/archive/1.9.11.tar.gz | tar xz
+RUN cd Gant-1.9.11; ./gradlew :gant:install
+
+# build sqlite4java
+RUN git clone -b linux-gccv4 https://bitbucket.org/almworks/sqlite4java.git
+RUN sed -i.org -e '/^platforms=/s/linux-i386, //' sqlite4java/ant/linux.properties
+RUN cd sqlite4java/ant; ~/lib/gant/bin/gant dist
+
+FROM alpine:3.7 AS jar
```

Comment:
These other containers don't really seem necessary to build the libsqlite4java binary.  I could see an argument that you're vetting that the binary is loaded properly by the runtime, but we proved last night that the runtime will start up just fine with the wrong binary.

Reply:
I think it is good to have there to test that dynamodb-local runs when everything is built. Plus, it will give us a starting spot if we ever want to use it in a container in the future.